testsuite/cluster-test/src/main.rs

```diff
@@ -148,6 +148,16 @@ pub fn main() {
             ));
             return;
         }
+    } else if args.health_check && args.swarm {
+        let mut util = BasicSwarmUtil::setup(&args);
+        let logs = DebugPortLogThread::spawn_new(&util.cluster).0;
+        let duration = Duration::from_secs(args.duration);
+        exit_on_error(run_health_check(
+            &logs,
+            &mut util.health_check_runner,
```

Comment:
Same question for `health_check_runner` - looks like it could be variable, not a field on `BasicSwarmUtil`?

Reply:
Yes, agree.